srcDoc/custom_types.rst

```diff
@@ -0,0 +1,578 @@
+.. _section-custom-types:
+
+============
+Custom Types
+============
+
+*RTI Perftest* allows the user to add their own `idl` types, instead of using
+the one provided by default. The purpose of this feature is to get a closer
+approximation to the performance behavior of the user's final system. The main
+reason why using one type or another will result in different outputs for latency
+and throughput is the serialization and deserialization times.
+
+This feature is designed to work for regular data, *DynamicData*
```

Comment:
This feature is designed to work for regular data, *DynamicData*
and *FlatData* type definitions (required to test Connext DDS FlatData and/or Shared Memory Zero Copy features).

Reply:
Done.